packages/shared/routes/dashboard/settings/views/help/Help.svelte

```diff
@@ -0,0 +1,26 @@
+<script lang="typescript">
+    import { HR } from 'shared/components'
+    import { HelpAndInfo } from 'shared/lib/typings/routes'
+    import { Discord, Documentation, FAQ, ReportAnIssue } from './'
+
+    const settings: {
+        component: unknown
+        childRoute: HelpAndInfo
+    }[] = [
+        { component: Documentation, childRoute: HelpAndInfo.Documentation },
+        { component: FAQ, childRoute: HelpAndInfo.FAQ },
+        { component: Discord, childRoute: HelpAndInfo.Discord },
+        { component: ReportAnIssue, childRoute: HelpAndInfo.ReportAnIssue },
```

Comment:
Each of these components are exactly the same maybe we can make 1 component and pass the data by props.

Reply:
refactored to single component